src/test/cypress/e2e/exam/ExamAssessment.cy.ts

```diff
@@ -151,6 +153,7 @@ describe('Exam assessment', () => {
 
         describe('Text exercise assessment', () => {
             beforeEach('Create exercise and submission', () => {
+                cy.login(admin);
```

Comment:
why is an admin login necessary here?
Creating exercises should done by instructors